src/common/misc.c

```diff
@@ -657,6 +657,12 @@ daos_crt_init_opt_get(bool server, int ctx_nr)
 	/** enable statistics on the server side */
 	daos_crt_init_opt.cio_use_sensors = server;
 
+	/** configure cart for maximum bulk threshold */
+	daos_crt_init_opt.cio_use_expected_size = true;
+	daos_crt_init_opt.cio_max_expected_size = DAOS_RPC_SIZE;
+	daos_crt_init_opt.cio_use_unexpected_size = true;
+	daos_crt_init_opt.cio_max_unexpected_size = DAOS_RPC_SIZE;
```

Comment:
just FYI, the unexpected_size and expected_size can be different if needed; but unexpected_size should be same at client and server, expected_size also be same for client and server.

Reply:
Thanks. Since we do inline for update and fetch operations, it sounds like we have to set both to DAOS_RPC_SIZE, correct?

Reply:
Ah, seems correct.

Reply:
indeed, we do need for reply